xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -20,8 +20,40 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents an assembly load context, which represents the runtime's concept of a load binding scope. This class is abstract and must be implemented to be instantiated.</summary>
```

Comment:
This and several other docs use the terms loading and binding interchangeably. But the term "load binding" is very confusing, because we bind the assembly (not the load itself) in the process of loading. That is, loading is the process of resolution and binding -- is my understanding of the terminology.